main.go

```diff
@@ -10,40 +11,47 @@ import (
 
 func main() {
 	// Needs as argument a path to an Antora project
-	if len(os.Args) < 2 {
-		fmt.Println("Please provide a path to an Antora project")
-		os.Exit(1)
-	}
-	path := os.Args[1]
+	var antoraPath string
+	flag.StringVar(&antoraPath, "antoraPath", "/docs", "Path to Antora document sources")
+
+	// Module to check. If none passed, it is assumed to be ROOT
+	var module string
+	flag.StringVar(&module, "module", "ROOT", "Module to analyze")
+
+	// File to parse and analyze; if none is passed, it is assumed to
+	// be a standard Antora navigation file at the ROOT module.
+	var filename string
+	flag.StringVar(&filename, "filename", "/modules/ROOT/nav.adoc", "File to analyze")
```

Comment:
No, because the tool does not always reference a nav.adoc for navigation; it could be a generic Asciidoc document like the one we use to generate the PDF version of the Handbook.